cfbs/commands.py

```diff
@@ -178,6 +179,12 @@ def init_command(index=None, masterfiles=None, non_interactive=False) -> int:
         do_git = True if do_git == "yes" else False
 
     if do_git is True:
+        try:
+            check_git_exists()
+        except FileNotFoundError as e:
+            print("Command 'git' was not found")
+            return 1
```

Comment:
Looks like this will never happen?

From what I can see in the function, `check_git_exists()` raises a `CFBSGitError`, not a `FileNotFoundError`.